### PR TITLE
Tidyup wallet_send

### PR DESCRIPTION
**Pull request by k9ert**
The wallet_send.html slowly became a 570+ LOC monster which is much better off by cutting it in different pieces separating the different aspects. With this PR we get:
* wallet_send.html - the send-form and only that form
* wallet_send_pending.html - the list of pending psbts
* wallet_send_sign_psbt - all the HW-wallet signing stuff

Apart from that this also includes a refactoring of the wallet_menu which is now located in a separate include-file.

### CODE REVIEW

File: src/cryptoadvance/specter/templates/wallet_sendpending.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	}
+</style>
+<h1>{{wallet.name}}</h1>
+<div class="row padded">
+	<a href="/wallets/{{wallet['alias']}}/tx/" class="btn radio left">Transactions</a>
+	<a href="/wallets/{{wallet['alias']}}/addresses/" class="btn radio">Addresses</a>
+	<a href="/wallets/{{wallet['alias']}}/receive/" class="btn radio">Receive</a>
+	<a href="/wallets/{{wallet['alias']}}/send/" class="btn radio checked">Send</a>
+	<a href="/wallets/{{wallet['alias']}}/settings/" class="btn radio right">Settings</a>
+</div>
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/templates/wallet_sendpending.html:45]: Should this also be replaced by `wallet_menu.html`?
```py
{% set active_menuitem = "send" %}
{% include "includes/wallet_menu.html" %}
```

**Comment by stepansnigirev** [comment]
Great refactoring!
I merged two previous pull requests, can you rebase and make sure the merged code doesn't dissapear? In particular, animated QR codes for full PSBT transaction.

**Comment by k9ert** [comment]
Rebased and force-pushed! I tried quite hard to not break anything. I've tested animated QR-codes.

File: src/cryptoadvance/specter/templates/wallet_settings.html [hunk 2]
```diff
@@ -10,13 +9,9 @@ <h1 id="nametitle">{{wallet.name}}</h1>
...
 	</div>
 </form>
-<div class="row padded">
-	<a href="/wallets/{{wallet['alias']}}/tx/" class="btn radio left">Transactions</a>
-	<a href="/wallets/{{wallet['alias']}}/addresses/" class="btn radio">Addresses</a>
-	<a href="/wallets/{{wallet['alias']}}/receive/" class="btn radio">Receive</a>
-	<a href="/wallets/{{wallet['alias']}}/send/" class="btn radio">Send</a>
-	<a href="/wallets/{{wallet['alias']}}/settings/" class="btn radio right checked">Settings</a>
-</div>
+<h1>{{wallet.name}}</h1>
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/templates/wallet_settings.html:12]: Duplicate of the wallet name, there is one in the form above already
Screenshot 2020-04-24 at 18 03 17

File: src/cryptoadvance/specter/templates/includes/wallet_menu.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<div class="row padded">
+	<a href="{{ url_for('wallet_tx',wallet_alias=wallet_alias)}}" class="btn radio left {% if active_menu_item == 'transactions' %}checked{% endif %}">Transactions</a>
+	<a href="{{ url_for('wallet_addresses',wallet_alias=wallet_alias)}}" class="btn radio {% if active_menu_item == 'addresses' %}checked{% endif %}">Addresses</a>
+	<a href="{{ url_for('wallet_receive',wallet_alias=wallet_alias)}}" class="btn radio {% if active_menu_item == 'receive' %}checked{% endif %}">Receive</a>
+	<a href="{{ url_for('wallet_send',wallet_alias=wallet_alias)}}" class="btn radio {% if active_menu_item == 'send' %}checked{% endif %}">Send</a>
+	<a href="{{ url_for('wallet_settings',wallet_alias=wallet_alias)}}" class="btn radio right {% if active_menu_item == 'settings' %}checked{% endif %}">Settings</a>
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/templates/includes/wallet_menu.html:6]: Here you compare against `active_menu_item` but everywhere you set `active_menuitem`
So active item is never highlighted

File: src/cryptoadvance/specter/templates/wallet_send_sign_psbt.html [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+	{% if wallet.uses_qrcode_device %}
+		import QrScanner from "/static/qr/qr-scanner.min.js";
+		QrScanner.WORKER_PATH = '/static/qr/qr-scanner-worker.min.js';
+
+		const video = document.getElementById('qr-video');
+
+		const scanner = new QrScanner(video, result => {
+			scanner.stop();
+			document.getElementById("popup").style.display = 'none';
+			{% if not psbt %}
```
Comment (stepansnigirev) [inline src/cryptoadvance/specter/templates/wallet_send_sign_psbt.html:276]: This part is used to scan destination address
So you need to copy QR scanning functionality to `wallet_send.html` as well
If we are in the send page - scanner should populate "address" field, if in PSBT - it should call combine function.

Comment (k9ert) [reply src/cryptoadvance/specter/templates/wallet_send_sign_psbt.html:276]: Good spot! Right now i don't have a Hardware-wallet to test the PSBT-part but the destination-address is now working. Might do that tonight or tomorrow.

**Comment by stepansnigirev** [comment]
Thanks for fixing everything :) 
Merging!

**Pull request merged**